aggie-coding-club/Rev-Registration
# Mobile improvements

## Description

Makes the website semi-functional on mobile. Not sure if this fixes scrolling on iOS though

## Rationale

Applied a bunch of CSS media queries

## How to test

Open the device toolbar in Chrome DevTools and mess around with a bunch of different devices. Key ones to check include: iPhone 5/SE (the smallest), iPhone X (which a lot of users will have), iPad both horizontal and vertical, because the breakpoint is set between the two), and also regular desktop to make sure nothing broke.

## Screenshots

See #522

## Related tasks

Closes #522
